services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceHelixAdmin.java

```diff
@@ -5719,6 +5728,11 @@ protected static HybridStoreConfig mergeNewSettingsIntoOldHybridStoreConfig(
             oldStore.getName() + " was not a hybrid store.  In order to make it a hybrid store both "
                 + " rewind time in seconds and offset or time lag threshold must be specified");
       }
+
+      String newRealTimeTopicName = VeniceSystemStoreUtils.isSystemStore(oldStore.getName())
```

Comment:
`VeniceSystemStoreUtils.isSystemStore(oldStore.getName())` change to `isFollowingNewNaming`